scripts/docgen/docgen.php

```diff
@@ -187,6 +187,32 @@ function save_file($filename, $content) { /* {{{ */
 }
 /* }}} */
 
+/** Returns an xml type tag or an entity to use in the generated xml for a PHP constant */
+function get_xml_type_tag_or_entity($value, bool $allow_entity) { /* {{{ */
+	// Entities from entities/global.ent are recommended for freeform text.
+	if (is_int($value)) {
+		return $allow_entity ? '&integer;' : '<type>int</type>';
+	} elseif (is_float($value)) {
+		return $allow_entity ? '&float;' : '<type>float</type>';
+	} elseif (is_null($value)) {
+		return $allow_entity ? '&null;' : '<type>null</type>';
```

Comment:
```suggestion
		return '<type>null</type>';
```
Ref. https://github.com/php/doc-base/blob/e3c21d3e44e43813e9c844b33747e3a68e18f596/entities/global.ent#L556-L559